flang/test/Lower/OpenMP/DelayedPrivatization/wsloop.f90

```diff
@@ -0,0 +1,42 @@
+! RUN: %flang_fc1 -emit-hlfir -fopenmp -mmlir --openmp-enable-delayed-privatization-staging \
```

Comment:
There may be an obvious thing I am missing here, but is there a reason for "staging" here? Last I did some tests on delayed privatisation, the flag was "openmp-enable-delayed-privatization", which sounded better.

Reply:
There are 2 flags. The staging one is for partial implementations of delayed privatization (i.e. where the relevant construct is not support across the entire pipeline; e.g. the MLIR to LLVM lowering is still missing like this case). The one you mentioned is for completely implemented constructs (like `omp.parallel`).

Reply:
Thanks. Understood. LGTM in that case.